docs/using-hive.md

```diff
@@ -188,21 +188,41 @@ oc get nodes
 
 Hive can optionally create delegated Route53 DNS zones for each cluster.
 
-To use this functionality you must first define the base domains you want to manage DNS for in your HiveConfig.
+To use this feature:
+
+  1. Manually create a Route53 hosted zone for your "root" domain (i.e. hive1.example.com in the example below) and ensure your DNS is operational.
+  1. Create a secret in the "hive" namespace with AWS credentials with permissions to manage the root hosted zone.
+     ```
+     apiVersion: v1
+     data:
+       aws_access_key_id: REDACTED
+       aws_secret_access_key: REDACTED
+     kind: Secret
+     metadata:
+       name: route53-aws-creds
+     type: Opaque
+     ```
+  1. Update your HiveConfig to enable externalDNS and set the list of managed domains:
+     ```
+     apiVersion: hive.openshift.io/v1alpha1
+     kind: HiveConfig
+     metadata:
+       name: hive
+     spec:
+       managedDomains:
+       - hive1.example.com
+       externalDNS:
+         aws:
+           credentials:
+             name: route53-aws-creds
+     ```
+
+You can now request clusters with clusterDeployment.spec.manageDNS=true and clusterDeployment.spec.baseDomain=mycluster.hive1.example.com. Hive will create a mycluster.hive1.example.com hosted zone and link it with your root hosted zone. Per-cluster DNS entries ("apps", "api", etc) will be created in the new mycluster.hive1.example.com hosted zone.
```

Comment:
What does it mean to "link" one hosted zone to another?

Reply:
see above description of hive creating a new route53 zone and then hive making the necessary NS entries in the managed domain

Reply:
PR updated to try to clarify the steps hive will perform, and the linking.